.gitignore

```diff
@@ -1,4 +1,5 @@
 /target
+Cargo.lock
```

Comment:
I'd prefer if `Cargo.lock` stays on git, just to make sure builds stay consistent.